apps/base/models.py

```diff
@@ -14,3 +19,28 @@ class TimeStampedModel(models.Model):
     class Meta:
         abstract = True
         app_label = 'base'
+
+
+"""
+The decorator below is used to pass model field names to django signals function.
+"""
+
+
+def extra_args(field_name, *args, **kwargs):
+    def extra_args_decorator(f, *args, **kwargs):
+        def extra_args_wrapper(sender, instance, **kwargs):
+            f(sender, instance, field_name=field_name, **kwargs)
+        return extra_args_wrapper
+    return extra_args_decorator
+
+
+"""
+The function below is used to log the change when any attribute of model field instance is changed.
```

Comment:
Can we shift this doc string inside the function? Also it will be great if we can update it to

```
when any model field value changes
```